service/create/service.go

```diff
@@ -582,15 +684,94 @@ func (s *Service) Boot() {
 
 					s.logger.Log("info", "deleted bucket objects")
 
-					lbInput := LoadBalancerInput{
-						Clients: clients,
-						Cluster: cluster,
+					// Delete API server Record Sets
```

Comment:
Please add a full stop at the end of the sentence. There are also other comments missing that. <3